src/java.desktop/share/classes/javax/swing/package-info.java

```diff
@@ -149,5 +149,7 @@
  * </ul>
  *
  * @serial exclude
+ *
+ * @since 1.1
```

Comment:
This isn't right. Where did you get this from ?
Swing only became part of the JDK in JDK 1.2
It was an unbundled library before then.
If you find any @since 1.1 tags in the Swing API they are a mistake.

Reply:
My bad then, this is a mistake.